app/src/main/kotlin/com/wire/android/ui/common/WireDropDown.kt

```diff
@@ -0,0 +1,157 @@
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.textfield.Label
+import com.wire.android.ui.common.textfield.WireTextFieldState
+import com.wire.android.ui.common.textfield.wireTextFieldColors
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+import com.wire.android.util.EMPTY
+import com.wire.kalium.logic.data.conversation.ConversationOptions
+
+@Composable
+internal fun WireDropDown(
+    items: List<String>, defaultItemIndex: Int = -1, label: String, modifier: Modifier, onChange: (selectedIndex: Int) -> Unit
+) {
+    var expanded by remember { mutableStateOf(false) }
+    var selectedIndex by remember { mutableStateOf(defaultItemIndex) }
+    val arrowRotation: Float by animateFloatAsState(if (expanded) 180f else 0f)
+    val placeHolderText = stringResource(R.string.wire_dropdown_placeholder)
+    val borderColor = MaterialTheme.wireColorScheme.secondaryButtonEnabledOutline
+    val shape = RoundedCornerShape(MaterialTheme.wireDimensions.textFieldCornerSize)
+    Column(modifier) {
+        Label(
+            label, false, WireTextFieldState.Default, remember { MutableInteractionSource() }, wireTextFieldColors()
+        )
+        Column(
+            modifier = Modifier.clip(shape).background(color = MaterialTheme.wireColorScheme.secondaryButtonEnabled, shape = shape)
+                .border(width = 1.dp, color = borderColor, shape)
+                .clickable {
+                    expanded = !expanded
+                }) {
+            Row(Modifier.padding(start = 16.dp, end = 16.dp, top = 12.dp, bottom = 12.dp)) {
+                Column(Modifier.weight(1f).align(Alignment.CenterVertically)) {
+                    Text(
+                        text = if (selectedIndex != -1) {
+                            items[selectedIndex] + GetDefaultTextIndicator(selectedIndex, defaultItemIndex)
+                        } else placeHolderText,
+                        style = MaterialTheme.wireTypography.body01,
+                        color = if (selectedIndex != -1) MaterialTheme.wireColorScheme.onSecondaryButtonEnabled
+                        else MaterialTheme.wireColorScheme.secondaryText
+                    )
+                }
+                Icon(
+                    imageVector = Icons.Filled.ExpandMore,
+                    contentDescription = stringResource(R.string.change),
+                    tint = MaterialTheme.wireColorScheme.secondaryText,
+                    modifier = Modifier.padding(top = 4.dp).rotate(arrowRotation)
+                )
+            }
+            if (expanded) Column {
+                AnimatedVisibility(
+                    visible = expanded, enter = fadeIn(animationSpec = tween(500, 200)) + expandVertically(
```

Comment:
Maybe it would be better to make const vals out of these duration millis (500, 400, 200)? 
Btw, I'm curious how did you determine these specific values? By trial and error? 😄